JitsiConferenceEventManager.js

```diff
@@ -437,6 +437,8 @@ JitsiConferenceEventManager.prototype.setupRTCListeners = function() {
         conference.room.connectionTimes['data.channel.opened'] = now;
         Statistics.analytics.sendEvent('conference.dataChannel.open',
             { value: now });
+
+        conference.eventEmitter.emit(JitsiConferenceEvents.DATA_CHANNEL_OPEN);
```

Comment:
They should go through the same abstraction. Both peerconnection and websocket go through BridgeChannel's _handleChannel(), which sets the channel.onopen callback, which should fire RTCEvents.DATA_CHANNEL_OPEN, which conference event manager reacts to.